app/backend/utils/dynamo/list_features.go

```diff
@@ -30,6 +28,8 @@ func (dc *DynamoConfig) ListFeatures(ctx context.Context, input *ListFeaturesInp
 		ProjectionExpression:      proj,
 		ExpressionAttributeNames:  exprAttrNames,
 		ExclusiveStartKey:         exclusiveStartKey,
+		ReturnConsumedCapacity:    types.ReturnConsumedCapacityTotal,
+		Limit:                     aws.Int32(25),
```

Comment:
two thoughts on this:
1. magic number are we abl to assign it a const or similar in GO? 
2. able to tie it to a env var instead that way if want to update it can change in env var instead of a redeployment?

Reply:
oh, hmmmm when did we kinda lose that benefit? thought that was always the benefit of env vars is that they didnt need a redeployment but i guess architectures change dont they haha

Reply:
With our infra, env vars are part of `app/deployments/lib/deployment-stack.ts` (infra single source of truth). You could change it in aws but you'd have to change it there too so it doesn't get overwritten the next deploy.